lib/cli.js

```diff
@@ -17,4 +17,5 @@ module.exports = CLI
   .option('-t, --tag <range>', 'generate from specific tag or range (e.g. v1.2.3 or v1.2.3..v1.2.4)')
   .option('-x, --exclude <types>', 'exclude selected commit types (comma separated)', list)
   .option('-f, --file [file]', 'file to write to, defaults to ./CHANGELOG.md, use - for stdout', './CHANGELOG.md')
-  .option('-u, --repo-url [url]', 'specify the repo URL for commit links, defaults to checking the package.json');
+  .option('-u, --repo-url [url]', 'specify the repo URL for commit links, defaults to checking the package.json')
+  .option('-s, --sub-directory <path>', 'specify an explicit relatvie path to a sub-directory');
```

Comment:
Can you make the description `specify a path to be passed into git log`? And then maybe add a subsection within `## Usage` in the README titled `### Sub-directory` under `### Recommended` to explain the nuances with how `--file` is relative from `--sub-directory` since it is a bit nuanced?